CONTRIBUTING.md

```diff
@@ -1,17 +1,27 @@
-2. 🔍 Search for issues tagged as [help wanted](https://github.com/learningequality/kolibri-design-system/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22+no%3Aassignee+) or [good first issue](https://github.com/learningequality/kolibri-design-system/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22+no%3Aassignee).
-3. 🗣️ Ask us for an assignment in the comments of an issue you've chosen.
-
-Note that in times of increased contributions activity, it may take us a few days to reply. If you don't hear from us within a week, please reach out via [Kolibri Design System GitHub Discussions](https://github.com/learningequality/kolibri-design-system/discussions).
+1. 💻 **Follow the [Getting started](./dev_docs/01_getting_started.md) to set up your development server.** At the bottom of the page, you can also find links that will take you to guidelines for the most common tasks.
+2. 🔍 **Search for issues tagged as [help wanted](https://github.com/learningequality/kolibri-design-system/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22+no%3Aassignee+) or [good first issue](https://github.com/learningequality/kolibri-design-system/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22+no%3Aassignee).**
+4. 🗣️ **Ask us for an assignment in the comments of an issue you've chosen.** Please request assignment of a reasonable amount of issues at a time. Once you finish your current issue or two, you are welcome to ask for more.
 
 **❓ Where to ask questions**
 
-- For anything development related, refer to the [developer documentation](./dev_docs/) at first. Some answers may already be there.
+- For anything development related, refer to the [Developer documentation](./dev_docs/) at first. Some answers may already be there.
 - For questions related to a specific issue or assignment requests, use the corresponding issue's comments section.
-- Visit [Kolibri Design System GitHub Discussions](https://github.com/learningequality/kolibri-design-system/discussions) to ask about anything related to contributing to Kolibri Design System, troubleshoot development server issues, or connect with other contributors.
+- Visit [GitHub Discussions](https://github.com/learningequality/kolibri-design-system/discussions) to ask about anything related to contributing or to troubleshoot development server issues.
+
+**👥 How to connect**
+
+- We encourage you to visit [GitHub Discussions](https://github.com/learningequality/kolibri-design-system/discussions) to connect with the Learning Equality team as well as with other contributors.
+- If you'd like to contribute on a regular basis, we are happy to invite you to our open-source community Slack channel. Get in touch with us at info@learningequality.org to receive an invitation.
+
+---
+
+🕖 Please allow us a few days to reply to your comments. If you don't hear from us within a week, reach out via [GitHub Discussions](https://github.com/learningequality/kolibri/discussions).
```

Comment:
i believe this link on line 21 should be updated as well to point to the KDS-specific discussions, unless i'm misunderstanding. 

i think there is also one last reference to the incorrect discussion board [on line 65 of the readme doc](https://github.com/learningequality/kolibri-design-system/pull/494/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R65), but once these are resolved i don't have any other nitpicks :)

Reply:
Oh, yes, you're right. It's not nitpicks, thank you.